aiida/engine/daemon/client.py

```diff
@@ -453,12 +443,116 @@ def stop_daemon(self, wait: bool) -> JsonDictType:
         return result
 
     def restart_daemon(self, wait: bool) -> JsonDictType:
-        """
-        Restart the daemon
+        """Restart the daemon.
 
-        :param wait: boolean to indicate whether to wait for the result of the command
-        :return: the client call response
+        :param wait: Boolean to indicate whether to wait for the result of the command.
+        :return: The client call response.
         """
         command = {'command': 'restart', 'properties': {'name': self.daemon_name, 'waiting': wait}}
-
         return self.call_client(command)
+
+    def start_daemon(self) -> None:
+        """Start the daemon in a sub process running in the background.
+
+        :raises DaemonException: If the daemon fails to start.
+        :raises DaemonException: If the daemon starts but then is unresponsive or in an unexpected state.
+        """
+        env = get_env_with_venv_bin()
+
+        try:
+            subprocess.check_output(self.cmd_start_daemon, env=env, stderr=subprocess.STDOUT)  # pylint: disable=unexpected-keyword-arg
+        except subprocess.CalledProcessError as exception:
+            raise DaemonException('The daemon failed to start.') from exception
+
+        time.sleep(1)
```

Comment:
Just wondering about this part here - is 1s always the appropriate time?

Does `get_status` return immediately or does it have a built-in timeout as well?

Reply:
Yep, you are absolutely right, the `get_status` method calls `call_client` which includes timeout functionality already, so this can be removed.